### PR TITLE
Update hello-hybrid-cert-trust-devreg.md

### PR DESCRIPTION
**Pull request by SanDeo-MSFT**

### CODE REVIEW

**Comment by SanDeo-MSFT** [comment]
#sign-off

**Comment by illfated** [comment]
The beginning of the sentence, "Both **adfs/services/trust/2005/windowstransport** or **adfs/services/trust/13/windowstransport** [ ... ]" looks a bit odd. I would think that "Both" should be replaced with "Either", unless the "or" between the 2 highlighted alternatives should be replaced with "and". Have I misunderstood something in that sentence?

**Comment by SanDeo-MSFT** [comment]
Thanks for the feedback. I will change it to and

Sent from my iPhone

On Aug 18, 2019, at 7:35 AM, Trond B. Krokli <notifications@github.com<mailto:notifications@github.com>> wrote:


The beginning of the sentence, "Both adfs/services/trust/2005/windowstransport or adfs/services/trust/13/windowstransport [ ... ]" looks a bit odd. I would think that "Both" should be replaced with "Either", unless the "or" between the 2 highlighted alternatives should be replaced with "and". Have I misunderstood something in that sentence?

**Comment by e0i** [comment]
@mapalko Please review the PR that has been introduced by a MS contributor. Thanks.

**Comment by e0i** [comment]
@mapalko Following up for updates.

**Comment by illfated** [comment]
Friendly reminder: The beginning of the sentence needs to be reviewed for possible misinterpretation, unless there is a more distant "and" later in the sentence which the initial "Both" refers to.

**Comment by mapalko** [comment]
I don't have any issues other than the "and" modification. @SanDeo-MSFT can you make the modification in your patch?

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ When you're using AD FS, you need to enable the following WS-Trust endpoints:
...
+> Both **adfs/services/trust/2005/windowstransport** or **adfs/services/trust/13/windowstransport** should be enabled as intranet facing endpoints only and must NOT be exposed as extranet facing endpoints through the Web Application Proxy. To learn more on how to disable WS-Trust Windows endpoints, see [Disable WS-Trust Windows endpoints on the proxy](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/deployment/best-practices-securing-ad-fs#disable-ws-trust-windows-endpoints-on-the-proxy-ie-from-extranet). You can see what endpoints are enabled through the AD FS management console under **Service** > **Endpoints**.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-devreg.md]:
```suggestion
> Both **adfs/services/trust/2005/windowstransport** and **adfs/services/trust/13/windowstransport** should be enabled as intranet facing endpoints only and must NOT be exposed as extranet facing endpoints through the Web Application Proxy. To learn more on how to disable WS-Trust Windows endpoints, see [Disable WS-Trust Windows endpoints on the proxy](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/deployment/best-practices-securing-ad-fs#disable-ws-trust-windows-endpoints-on-the-proxy-ie-from-extranet). You can see what endpoints are enabled through the AD FS management console under **Service** > **Endpoints**.
```
(verified by @mapalko)

**Comment by DaniHalfin** [comment]
The default branch for this repository has changed. Commenting to remember to merge this to the new default, if this PR gets merged.

**Comment by e0i** [comment]
@SanDeo-MSFT 

The default branch for this repository has changed from `master` to `public`. Please edit your pull request to change the merge target to `public` via the `Edit` button at the top right corner. Thanks.

**Comment by e0i** [comment]
@SanDeo-MSFT Would you also be able to follow-up with revision suggestions provided by the document author? We would like to proceed with merge. Thanks.

**Comment by Dansimp** [comment]
made copyedit change directly in branch. Merging

**Pull request merged**